src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -776,13 +779,18 @@ def start_servers(self, server_groups=None):
 
         if isinstance(server_groups, dict):
             for group, info in list(server_groups.items()):
+                self.log.debug(
+                    "## test.py - start_servers 2. group = {}".format(group))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions